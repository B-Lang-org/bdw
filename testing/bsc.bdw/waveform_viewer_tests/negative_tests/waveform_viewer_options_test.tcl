
# Specifies options for waveform viewer
handlers::set_waveform_viewer 

# Specifies Non-Bsv hierarchy for the waveform viewer
handlers::set_nonbsv_hierarchy

# Opens the specified project
handlers::open_project project.bspec 

# Specifies options for waveform viewer
handlers::set_waveform_viewer 

# Specifies options for waveform viewer
handlers::set_waveform_viewer wrong 

# Specifies options for waveform viewer
handlers::set_waveform_viewer Novas -comm

# Specifies options for waveform viewer
handlers::set_waveform_viewer Novas -command

# Specifies options for waveform viewer
handlers::set_waveform_viewer Novas -command command -options option -close yes

# Specifies Non-Bsv hierarchy for the waveform viewer
handlers::set_nonbsv_hierarchy

# Closes the current project
handlers::close_project

# Exits the workstation
exit
