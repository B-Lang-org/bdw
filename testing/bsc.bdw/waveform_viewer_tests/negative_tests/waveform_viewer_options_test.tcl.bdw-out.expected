There is no open project
There is no open project
The Waveform Viewer is not specified
wrong is not supported viewer
Unknown option: -comm
Argument -command expects an argument none found
The argument for -close is invalid
 should be 0 or 1
The directory is not specified
